applications/nrf_desktop/README.rst

```diff
@@ -28,6 +28,16 @@ This means that parts of application functionality were separated into isolated
 Application events are handled by the :ref:`event_manager`.
 Modules register themselves as listeners of events they need to react on.
 
+The module that reacts on given application event is called a sink module.
+The module that submits given application event is called a source module.
+The application event can be submitted by multiple modules and it can have multiple listeners.
+The event-based communication for an application module is presented in an array.
+For an example, see :ref:`nrf_desktop_passkey` documentation.
```

Comment:
Please present this as a table, but instead of module names, add explanation in the cells.